free5gc-operator/api/v1alpha1/smfdeployment_types.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2023.
```

Comment:
Copyright 2023 The Nephio Authors

Reply:
Yes this PR needs changes in line with https://github.com/nephio-project/api/pull/11